### PR TITLE
Remove geom data

### PR DESCRIPTION
**Pull request by xylar**
This data can now be included in the `geometric_features` conda package, and the `compass_0.1.0` environment includes the version with this data.

### CODE REVIEW

**Comment by xylar** [comment]
I'm having trouble updating compass 0.1.0 on cori at the moment because of some file system issue.  We don't have a compass environment there at all at the moment.  Everywhere else (LANL IC, compy and blues) we have environments that include geometric_data in them, so we no longer need to point to a local cache of the data.

File: testing_and_setup/compass/README_ocean.md [hunk 1]
```diff
@@ -6,21 +6,25 @@ To set up and run ocean test cases from COMPASS, you will need a conda
 environment.  First, install Miniconda3 (if miniconda is not already
 installed), then create a new conda environment as follows:
 ``` bash
-conda create -n compass_py3.7 -c conda-forge -c xylar python=3.7 \
-    geometric_features=0.1.4 mpas_tools=0.0.6 jigsaw=0.9.11 jigsawpy=0.0.2 \
-    metis pyflann scikit-image basemap pyamg ffmpeg
+conda create -n compass_0.1.0 -c conda-forge -c e3sm python=3.7 compass=0.1.0
```
Comment (mark-petersen) [inline testing_and_setup/compass/README_ocean.md]: This is now out of date. I'll just use the second commit, as the first commit updated the conda version.

**Comment by mark-petersen** [comment]
Kept second commit, removing geometric_data links.

File: testing_and_setup/compass/README_ocean.md (hunk 1 above)
Comment (xylar) [reply testing_and_setup/compass/README_ocean.md]: Correct.  This is now taken care of in the `compass` metapackage.

**Pull request merged**